doc/memos/rdm-draft-alamos-ieee802154-radio-hal.md

```diff
@@ -848,59 +848,41 @@ typedef enum {
 A 802.15.4 HAL implementation MUST indicate all capabilities supported by the
 device and driver implementation.
 
-# 7 Future proof considerations
+# 7 Future Proof Considerations
 
-The Radio HAL is designed to be agnostic to the 802.15.4 MAC version on top.
-However, it is important to mention that most radio devices are optimized for a
-single version of the standard. In practice, hardware accelerations are usually
-not interchangeable between different versions of 802.15.4. As an example,
-devices that support frame retransmissions for IEEE802.15.4 2006 don't support
-Enhanced Beacons required by the TSCH layer of the 2012 standard.
+The Radio HAL is designed to be agnostic to different versions of the IEEE802.15.4 standard.
+A single radio device typically implements hardware acceleration for only one standard,
+whereas different standards are not always compatible. As an example, IEEE802.15.4--2006
+devices do not support Enhanced Beacons which is required by the TSCH layer of IEEE802.15.4--2012.
```

Comment:
My fault, this should be "Enhanced Acknowledgement" (I wrote the wrong term during the offline discussion).